lib/steam_valve/steam_user_stats.rb

```diff
@@ -0,0 +1,8 @@
+require 'httparty'
+
+module SteamValve
+  def self.current_players(game_id)
+    @api_endpoint_url =  "https://api.steampowered.com/ISteamUserStats/GetNumberOfCurrentPlayers/v1/?appid=#{game_id}"
+    HTTParty.get(@api_endpoint_url).to_hash
```

Comment:
This line has been used in a few places now. Maybe put it into a method and take the url as an argument?

Something like

```
def http_get(url)
    HTTParty.get(url).to_hash
end
```
Something like that will DRY up your code and also, if the implementation changes (say you dont want to convert it to a hash anymore) you only need to change it in one place.